blog/2024-05-16-blog-status-of-lbse-in-webkit.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Status of LBSE in WebKit"
```

Comment:
Using LBSE in the title and early without definition makes a lot of potential audience unlikely to click and read probably. I know it is long, but "Layer Based SVG Engine" is what I would use in the title - otherwise, people interested in SVG are unlikely to read it, right?

Reply:
I concur.

Reply:
I tried to do a compromise, i.e. spell it out but also mention LBSE, as a lot of people are familiar with that acronym now :)

Reply:
Yeah, it's fine to use LBSE once you introduce it in the same piece - but the title especially is the one that was going to grab or not - if you don't already know it, it wouldn't grab you.

Reply:
I agree with @meyerweb and @bkardell here, I think we should avoid using LBSE in the title and go for something more descriptive. See for instance what @nikolaszimmermann did in the previous post at https://wpewebkit.org/blog/05-new-svg-engine.html